etl/app/server/profiles/actions.js

```diff
@@ -35,6 +36,73 @@ export const createQuery = `CREATE TABLE IF NOT EXISTS ${tableName}
     includeinmeasure VARCHAR(1)
   )`;
 
+export const createIndexes = `
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_actionagency_asc
+    ON ${tableName} USING btree
+    (actionagency COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_actionid_asc
+    ON ${tableName} USING btree
+    (actionid COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_actionname_asc
+    ON ${tableName} USING btree
+    (actionname COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_assessmentunitid_asc
+    ON ${tableName} USING btree
+    (assessmentunitid COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+  CREATE INDEX IF NOT EXISTS ${indexTableName}_assessmentunitname_asc
+    ON ${tableName} USING btree
+    (assessmentunitname COLLATE pg_catalog."default" ASC NULLS LAST)
+    TABLESPACE pg_default;
+
```

Comment:
The `completiondate` index is missing:
```suggestion
CREATE INDEX IF NOT EXISTS ${indexTableName}_completiondate_desc
    ON ${tableName} USING btree
    (completiondate COLLATE pg_catalog."default" DESC NULLS LAST)
    TABLESPACE pg_default;
```